spec/controllers/idv/in_person/usps_locations_controller_spec.rb

```diff
@@ -93,6 +85,28 @@
       allow(UspsInPersonProofing::Proofer).to receive(:new).and_return(proofer)
     end
 
+    context 'with EIPP enabled' do
+      let(:vtr) { ['C1.C2.P1.Pe'] }
+      let(:eipp_sp_session) { { vtr: vtr, acr_values: nil } }
+      let(:user) { build(:user) }
+      let(:sp) { build(:service_provider, ial: 2) }
+
+      before do
+        allow(controller).to receive(:sp_session).and_return(eipp_sp_session)
+        allow(controller).to receive(:sp_from_sp_session).and_return(sp)
+      end
+
+      it 'sends the proofer the correct vector of trust' do
+        expect(AuthnContextResolver).to receive(:new).with(
+          user: user, service_provider: sp,
+          vtr: vtr, acr_values: nil
+        ).and_call_original
```

Comment:
I feel like stubbing and validating `.new` and calling original, what does this help with? The important part is the `true` argument to `request_facilities`?